extensions/store/sql/identity-hub-credentials-store-sql/src/main/java/org/eclipse/edc/identityhub/store/sql/credentials/SqlCredentialStoreExtension.java

```diff
@@ -34,8 +35,12 @@
 public class SqlCredentialStoreExtension implements ServiceExtension {
     public static final String NAME = "CredentialResource SQL Store Extension";
 
+    @Deprecated(since = "0.8.1")
     @Setting(value = "Datasource name for the DidResource database", defaultValue = DataSourceRegistry.DEFAULT_DATASOURCE)
     public static final String DATASOURCE_SETTING_NAME = "edc.datasource.credentials.name";
+    @Setting(value = "The datasource to be used", defaultValue = DataSourceRegistry.DEFAULT_DATASOURCE)
```

Comment:
nit: i would call the deprecated property `DATASOURCE_SETTING_NAME_DEPRECATED`, and the new one `DATASOURCE_SETTING_NAME`

Reply:
unfortunately I didn't do that in other repositories so I this point I would just let it like this, anyway I think that the `@Deprecated` annotation should be enough